themes/ddbasic/sass/configuration/_extend.scss

```diff
@@ -0,0 +1,133 @@
+//
+// Extends
+
+// ==========================================================================
+// Button
+// ==========================================================================
+%button {
+  @include transition(
+    background-color $speed $ease,
+    color $speed $ease
+  );
+  display: block;
+  border-radius: $round-corner;
+  padding: 20px 80px 14px 15px;
+  background-color: $grey;
+  color: $charcoal;
+  // Hover for non touch devices
+  .no-touch & {
+    &:hover {
+      background-color: $color-secondary;
+      color: $color-text-on-secondary;
+    }
+  }
+
+  @include media($mobile) {
+    padding: 20px 20px 14px 15px;
+  }
+}
+
+// ==========================================================================
+// Icon extentions
+// ==========================================================================
+
+// Icons
+%icon-arrow-down-small {
+  content: "\e900";
+}
+
+%icon-arrow-down {
+  content: "\e901";
```

Comment:
These extends are very useful in our icon mixins. And we believe the naming of these extends are semantic in the same way a variable would be.